sgf-viewer/src/content.ts

```diff
@@ -1037,10 +1101,62 @@ export const pages: object =
+                        "line": 6
+                    },
+                    {
+                        "path": "/shared/match/paper-robustness/iterated-adversarial-training/attack-ft-h9/vary-victim-visits/0/sgfs/F8BD6EC8E34E731E.sgfs",
+                        "line": 3
+                    },
+                    {
+                        "path": "/shared/match/paper-robustness/iterated-adversarial-training/attack-ft-h9/vary-victim-visits/0/sgfs/69420E7F02BD6F34.sgfs",
+                        "line": 1
+                    },
+                    {
+                        "path": "/shared/match/paper-robustness/iterated-adversarial-training/attack-ft-h9/vary-victim-visits/0/sgfs/1F4C87C39A93297B.sgfs",
+                        "line": 2
+                    },
+                    {
+                        "path": "/shared/match/paper-robustness/iterated-adversarial-training/attack-ft-h9/vary-victim-visits/0/sgfs/66691A5DE7E9F14B.sgfs",
+                        "line": 2
+                    },
+                    {
+                        "path": "/shared/match/paper-robustness/iterated-adversarial-training/attack-ft-h9/vary-victim-visits/0/sgfs/4486B179AC67B269.sgfs",
+                        "line": 8
+                    },
+                    {
+                        "path": "/shared/match/paper-robustness/iterated-adversarial-training/attack-ft-h9/vary-victim-visits/0/sgfs/6101158ACA9ECD02.sgfs",
+                        "line": 4
+                    },
+                    {
+                        "path": "/shared/match/paper-robustness/iterated-adversarial-training/attack-ft-h9/vary-victim-visits/0/sgfs/354CAC78BB59267D.sgfs",
+                        "line": 1
+                    },
+                    {
+                        "path": "/shared/match/paper-robustness/iterated-adversarial-training/attack-ft-h9/vary-victim-visits/0/sgfs/F835D8246453F61E.sgfs",
+                        "line": 6
+                    },
+                    {
+                        "path": "/shared/match/paper-robustness/iterated-adversarial-training/attack-ft-h9/vary-victim-visits/0/sgfs/4FF021B5FEBD34FA.sgfs",
+                        "line": 8
+                    }
+                ],
+                "max_games": 10,
+                "adversary": "<code>stall-adversary</code>",
+                "victim": "<code>v</code><sub><code>9</code></sub>, 512 visits",
+                "description": [
+                    "We trained another adversary by fine-tuning a cyclic adversary. Again it was able to defeat <code>v</code><sub><code>9</code></sub> at 512 visits of search convincingly, achieving a win rate of 91.5%, but the win rate is 5% at 4096 visits. The adversary stalls in the first few moves by passing, so we call it <code>stall-adversary</code>. (If the opponent is white they can pass as well to end the game and win by komi, but one of KataGo's default settings <code>conservativePass</code> disallows this.)"
```

Comment:
```suggestion
                    "We trained another adversary by fine-tuning a cyclic adversary. Again it was able to convincingly defeat <code>v</code><sub><code>9</code></sub> at 512 visits of search, achieving a win rate of 91.5%, with the win rate dropping to 5% at 4096 visits. The adversary stalls in the first few moves by passing, so we call it <code>stall-adversary</code>. (When the victim is white it could pass after the adversary passes to end the game and win by komi, but one of KataGo's default settings <code>conservativePass</code> disallows this.)"
```